_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -1703,7 +1703,13 @@ void TaskManager::UpdatePTS_Mode30i60p(
     mfxU32 taskIndex,
     mfxStatus *intSts)
 {
-    if (input == NULL)
+    if (output == nullptr)
+    {
+        *intSts = MFX_ERR_MORE_SURFACE;
```

Comment:
I'm not sure that this is correct status in the current case.

I mean : if we didn't check this parameter for nullptr previously and dereference in below in several places - maybe in this case "output" always not nullptr? So, we if it can be null maybe we need to return status MFX_ERR_UNDEFINED_BEHAVIOR or MFX_ERR_NULL_PTR of throw an exception ?

Please, clarify it.

Reply:
Look at line 1391. In case of SimpleMode we check that input and output is not equal to null, and I think that in case of Mode30i60p we should do the same check. 

About return MFX_ERR_MORE_SURFACE. At line 1294: if mode 30i60p is not enabled then we may call DoCpuFRC_AndUpdatePTS where in case of output == null we return MFX_ERR_MORE_SURFACE. I suppose that in case of mode 30i60p we should return same status.

Reply:
@DenWolf are you ok with this change?